common/src/main/java/org/apache/celeborn/common/network/util/NettyUtils.java

```diff
@@ -150,7 +150,8 @@ public static PooledByteBufAllocator getPooledByteBufAllocator(
           allowCache && conf.getCelebornConf().networkMemoryAllocatorAllowCache());
     }
     PooledByteBufAllocator allocator =
-        createPooledByteBufAllocator(conf.preferDirectBufs(), allowCache, conf.clientThreads());
+        createPooledByteBufAllocator(
+            conf.preferDirectBufs(), allowCache, conf.getCelebornConf().networkAllocatorArenas());
```

Comment:
The default value of `conf.clientThreads()` is zero which means that Celeborn might have been running with 0 arenas for a period of time.

Reply:
According to this code, zero will be turn into core nums, which is `conf.getCelebornConf().networkAllocatorArenas()` default value for now.
```
  private static PooledByteBufAllocator createPooledByteBufAllocator(
      boolean allowDirectBufs, boolean allowCache, int numCores) {
    if (numCores == 0) {
      numCores = Runtime.getRuntime().availableProcessors();
    }
    return new PooledByteBufAllocator(
        allowDirectBufs && PlatformDependent.directBufferPreferred(),
        Math.min(PooledByteBufAllocator.defaultNumHeapArena(), numCores),
        Math.min(PooledByteBufAllocator.defaultNumDirectArena(), allowDirectBufs ? numCores : 0),
        PooledByteBufAllocator.defaultPageSize(),
        PooledByteBufAllocator.defaultMaxOrder(),
        allowCache ? PooledByteBufAllocator.defaultSmallCacheSize() : 0,
        allowCache ? PooledByteBufAllocator.defaultNormalCacheSize() : 0,
        allowCache && PooledByteBufAllocator.defaultUseCacheForAllThreads());
  }
```

Reply:
Thanks for this explanation. As the default values is not changed. I'll merge this PR.